### PR TITLE
VACMS-12073: Add new util function for checking an invalid search string

### PR DESCRIPTION
**Pull request by chriskim2311**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
For this PR we take a new ask for checking for blank space search strings and add to existing logic for str character limits. We are now using a util function to handle all the logic in the search components.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12073

## Testing done
Tested the new logic against the different str requirements in the search components. 
## What areas of the site does it impact?

Search

### CODE REVIEW

File: src/applications/search/utils.js [hunk 1]
```diff
@@ -16,3 +16,8 @@ export function truncateResponseString(string, maxLength) {
 export function removeDoubleBars(string) {
   return string.replace('| Veterans Affairs', '');
 }
+
+export function isSearchStrInvalid(str) {
+  const formattedStr = str.trim();
+  return !formattedStr.length > 0 || !formattedStr.length > 255;
```
Comment (randimays) [inline src/applications/search/utils.js:22]: Seems like we could use 

```!formattedStr.length || formattedStr.length > 255```

Isn't `!formattedStr.length > 255` actually valid input? I tested in the review instance and it seems to be working but I'm not sure why.

Comment (chriskim2311) [reply src/applications/search/utils.js]: Thanks Randi. This is a better solution. This util function is only used for the call to the API. We still have the length string logic in place from before, so the correct error message will display for the long string length.

**Comment by jilladams** [comment]
Re: spaces searches, approved copy from Jordan: 

Your search didn't go through
Enter a search term that contains letters or numbers to find what you're looking for.

**Review by randimays — APPROVED** [review]
LGTM

**Pull request merged**